### PR TITLE
JS SDK init version.

### PR DESCRIPTION
**Pull request by yiyione**
The init version of the `OpenPAI javascript SDK`.
#3401

### CODE REVIEW

File: contrib/js-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
+# OpenPAI JS SDK
+
+The `Javascript` SDK for `OpenPAI`.
```
Comment (abuccts) [inline contrib/js-sdk/README.md]: `Javascript` -> `JavaScript`

File: contrib/js-sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+## Installation
+
+```bash
+npm install --save npm install yiyione/pai#openpai-js-sdk
```
Comment (abuccts) [inline contrib/js-sdk/README.md]: duplicate `npm install`

File: contrib/js-sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+```
+
+## Restful APIs
```
Comment (abuccts) [inline contrib/js-sdk/README.md]: RESTful

File: contrib/js-sdk/src/client/authnClient.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License in the project root for license information.
+ * @author Microsoft
+ */
```
Comment (abuccts) [inline contrib/js-sdk/src/client/authnClient.ts]: use the full license header?

File: contrib/js-sdk/tests/common/test_data/testAllVirtualClusters.ts [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+            "GPUs": 3
+        },
+        "status": "RUNNING",
+        "dedicated": false,
+        "resourcesTotal": {
+            "vCores": 85.8000114,
+            "memory": 740295.209472,
+            "GPUs": 14.3000019
+        },
+        "nodeList": [
```
Comment (abuccts) [inline contrib/js-sdk/tests/common/test_data/testAllVirtualClusters.ts]: change to some fake addresses?

File: contrib/js-sdk/README.md (hunk 1 above)
Comment (yiyione) [reply contrib/js-sdk/README.md]: Done.

File: contrib/js-sdk/README.md (hunk 2 above)
Comment (yiyione) [reply contrib/js-sdk/README.md]: Done.

File: contrib/js-sdk/README.md (hunk 3 above)
Comment (yiyione) [reply contrib/js-sdk/README.md]: Done.

File: contrib/js-sdk/src/client/authnClient.ts (hunk 4 above)
Comment (yiyione) [reply contrib/js-sdk/src/client/authnClient.ts]: Done.

File: contrib/js-sdk/tests/common/test_data/testAllVirtualClusters.ts (hunk 5 above)
Comment (yiyione) [reply contrib/js-sdk/tests/common/test_data/testAllVirtualClusters.ts]: Done.

**Pull request closed without merging**